can-view-callbacks.js

```diff
@@ -6,11 +6,78 @@ var domMutate = require('can-dom-mutate/node');
 var namespace = require('can-namespace');
 var nodeLists = require('can-view-nodelist');
 var makeFrag = require("can-util/dom/frag/frag");
+var canSymbol = require("can-symbol");
 
 //!steal-remove-start
 var requestedAttributes = {};
 //!steal-remove-end
 
+var tags = {};
+
+var GLOBAL = getGlobal();
```

Comment:
Don't cache this. Even thought this feature will (likely) never run in SSR, having this here might mean someone else uses it in code that will.